staging/src/k8s.io/apiserver/pkg/cel/library/format.go

```diff
@@ -193,7 +204,7 @@ var ConstantFormats = map[string]apiservercel.Format{
 		},
 		// Use govalidator url regex to estimate, since ParseRequestURI
 		// doesnt use regex
-		MaxRegexSize: len(govalidator.URL),
```

Comment:
switch this to a commented constant for the current length of this URL string... no need to import all the dynamic regex string calculation

Reply:
Will switch to constants!

https://go.dev/play/p/axz7eIgf4DS

```
length of Base64: 84
length of URL: 1103
```